### DISCUSSION TITLE
Expedition 5: Sightseer milestone issues

### DISCUSSION THREAD

**Discussion by eSonOfAnder** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#23] [Q&A]
I have a sneaking suspicion I know what the problem is here, but I want to check here first. I am attempting to run through the Expedition 5 Redux, and when I get to the Sightseer milestone the game is not registering when I land on a planet with one of the required biomes. My suspicion is that, like with Expedition 2, the "visit" flag only counts when you visit a planet for the first time, and is invalidated by using the Planetary Probe on your freighter. I am working to find some of the required biomes in neighboring systems now and not use the Planetary Probe to see if this does fix the issue, and will report back once I've been able to confirm.

**Comment by cwmonkey** [comment]
Yep, and I believe the issue is mentioned in the readme.

**Reply by eSonOfAnder** [reply]
It's not, I checked there first.

Edit: It's not in the readme for Expedition 5; it is there for Expedition 2.

**Reply by cwmonkey** [reply]
@leodium ^
